_content/libraries.md

```diff
@@ -1,3 +1,13 @@
+## Wallets
```

Comment:
```suggestion
## Testnet Wallets
```

That'd help differentiate it from any live Interledger wallets (Stronghold, GateHub, tip bot, etc)